source/guides/ember-cli/index.md

```diff
@@ -0,0 +1,56 @@
+Ember comes with a toolchain called Ember CLI that provides everything you need while developing your app behind a single command:  ```ember```.
+
+* State of the art asset management (use ```ember build``` to build your entire project)
+* Quick scaffolding like ```ember generate route some-page```
+* ES6 modules to keep your project organized
```

Comment:
I think @wycats would prefer if we called these something like "native JavaScript modules" or something, since ES6 has been rebranded as ES2015 and these are just part of the language spec at this point. May want this to link to [jsmodules.io](http://jsmodules.io) also.